### ISSUE TITLE
Handle OSError when getting jinja template

### ISSUE THREAD

**Issue by The-Compiler** [The-Compiler/qutebrowser#1466]
Similar to `jinja2.exceptions.UndefinedError` we could also handle `OSError` in `utils.jinja.render` and display an appropriate error page.

**Comment by The-Compiler** [comment]
Oh, the exception actually happens in `get_source`:

```
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/qutebrowser/utils/jinja.py", line 48, in get_source
  File "/usr/lib/python3.5/site-packages/qutebrowser/utils/utils.py", line 97, in read_file
  File "/usr/lib/python3.5/site-packages/pkg_resources/__init__.py", line 1184, in resource_string
  File "/usr/lib/python3.5/site-packages/pkg_resources/__init__.py", line 1457, in get_resource_string
  File "/usr/lib/python3.5/site-packages/pkg_resources/__init__.py", line 1580, in _get
OSError: [Errno 24] Too many open files: '/usr/lib/python3.5/site-packages/qutebrowser/html/error.html'
 
During handling of the above exception, another exception occurred:
 
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/qutebrowser/browser/webpage.py", line 496, in extension
  File "/usr/lib/python3.5/site-packages/qutebrowser/browser/webpage.py", line 174, in _handle_errorpage
  File "/usr/lib/python3.5/site-packages/qutebrowser/utils/jinja.py", line 89, in render
  File "/usr/lib/python3.5/site-packages/jinja2/environment.py", line 812, in get_template
  File "/usr/lib/python3.5/site-packages/jinja2/environment.py", line 774, in _load_template
  File "/usr/lib/python3.5/site-packages/qutebrowser/utils/jinja.py", line 50, in get_source
jinja2.exceptions.TemplateNotFound: error.html
```

**Comment by The-Compiler** [comment]
Fixed in #2192.
